src/pages/create-option.tsx

```diff
@@ -39,10 +39,6 @@ function CreateOption() {
     const [assets, setAssets] = useState<NFTAsset[]>([dummyNFT]);
     const [formState, setFormState] = useState<FormState>({
         asset: dummyNFT,
```

Comment:
```suggestion
        asset: undefined,
```

Reply:
will conflict with a fix i did

Reply:
If undefined, we need to fix :

![image](https://user-images.githubusercontent.com/49437873/170783207-25ea92eb-4d9d-4ecd-b64c-56e5970d6b91.png)

Reply:
my PR fixes this